src/Files.App/Contexts/ContentPage/IContentPageContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Files.App.Filesystem;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Files.App.Contexts
+{
+	public interface IContentPageContext : INotifyPropertyChanged
```

Comment:
What is this interface for?

Reply:
This interface is a context service which gives the active content of the page (The type of page, the selected elements, ...).
The properties of this context are useful to most commands and must be observable for the command to notify its state changes.

If I want to know the selected items, currently I have to observe ToolbarViewModel, but this instance itself can change
without notification if I switch tabs. It is therefore necessary to observer several "floors" of an object and update each time the floor changes. The context makes its information easy to obtain and notifies of changes.

Reply:
I guess it's okay, but on the other hand it bothers me because we're just further spreading the use of IShellPage...

Reply:
What do you mean ?

Reply:
@d2dyno1 is this good to go?